index.html

```diff
@@ -38,54 +39,54 @@ <h2 class="headh2">THE BEST FOOD BLOG ON THE WEB</h2>
+
+            <p>Pancetta turducken fatback cupim alcatra corned beef, ribeye pig tongue ham hock sirloin flank. Alcatra sausage doner spare ribs, turducken venison tri-tip meatloaf ham hock. Frankfurter alcatra pork belly picanha, spare ribs ball tip jowl beef ribs short ribs chuck hamburger turkey venison ribeye. Leberkas turducken venison, rump meatloaf strip steak ball tip chuck chicken kielbasa flank pig ribeye pork chop.</p>
+
+            <div class="ingredback">
+              <h2 class="hidden">Ingredients</h2>
+              <form class="left listleft">
+                <input id="Option" type="checkbox" autocomplete="off"><label class="chk" for="Option">1 1/2 cups milk</label><br>
+                <input id="Option2" type="checkbox" autocomplete="off"><label class="chk" for="Option2">1/2 cup mascarpone</label><br>
+                <input id="Option3" type="checkbox" autocomplete="off"><label class="chk" for="Option3">1/2 tsp pink salt</label><br>
+                <input id="Option4" type="checkbox" autocomplete="off"><label class="chk" for="Option4">1 lb Black Mission figs</label><br>
+                <input id="Option5" type="checkbox" autocomplete="off"><label class="chk" for="Option5">1/2 cup brown sugar</label><br>
+                <input id="Option6" type="checkbox" autocomplete="off"><label class="chk" for="Option6">2-4 tbsp water</label><br>
+              </form>
+              <form class="right listright">
+                <input id="Option7" type="checkbox" autocomplete="off"><label class="chk" for="Option7">1 1/2 cups heavy cream</label><br>
+                <input id="Option8" type="checkbox" autocomplete="off"><label class="chk" for="Option8">1/3 cup granulated sugar</label><br>
+                <input id="Option9" type="checkbox" autocomplete="off"><label class="chk" for="Option9">2 egg yolks</label><br>
+                <input id="Option10" type="checkbox" autocomplete="off"><label class="chk" for="Option10">1 lemon, juiced</label><br>
+                <input id="Option11" type="checkbox" autocomplete="off"><label class="chk" for="Option11">2 tbsp butter</label><br>
+                <input id="Option12" type="checkbox" autocomplete="off"><label class="chk" for="Option12">1 cup honey roasted pecan, roughly chopped</label><br>
+              </form>
+            </div>
           </article>
 
         <div class="break"></div>
-
-          <img src="imgs/headshot.png" alt="vanessa headshot" class="left headshot">
-          <h3>Vanessa Stevenson</h3>
-          <p class="sharep">Food enthusiast, photography fan. Add a pinch of raw foodism and that's pretty much who I am.</p>
+          <aside>
+            <img src="imgs/headshot.png" alt="vanessa headshot" class="left headshot">
+            <h3>Vanessa Stevenson</h3>
+            <p class="sharep">Food enthusiast, photography fan. Add a pinch of raw foodism and that's pretty much who I am.</p>
+          </aside>
```

Comment:
Dexter would argue the aside would be a 'section' or 'footer' as he said earlier in class. However, that's why it's called HTML5 semantics and we're literally arguing over semantics.